package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "protobufjs",
-  "version": "6.8.8",
+  "version": "7.0.0",
```

Comment:
@alexander-fenster Now that the release-please bot is handling version bumps, how should this be handled?

Reply:
release-please now supports monorepos.

Cc: @bcoe